### PR TITLE
Feature/118 delete test activities

### LINKED ISSUE #118: Delete activities that are generated during tests

### PR DESCRIPTION
**Pull request by bitshrine**

### CODE REVIEW

File: app/src/androidTest/java/com/sdp/movemeet/HomeScreen/HomeScreenActivityTest.java [hunk 1]
```diff
@@ -22,18 +30,65 @@
 @RunWith(AndroidJUnit4.class)
 public class HomeScreenActivityTest {
 
-    @Rule
-    public ActivityScenarioRule<HomeScreenActivity> HomeScreenTestRule = new ActivityScenarioRule<>(HomeScreenActivity.class);
+    /*@Rule
```
Comment (RoxaneBurri) [inline app/src/androidTest/java/com/sdp/movemeet/HomeScreen/HomeScreenActivityTest.java:33]: why do you comment this rule ?

File: app/src/main/java/com/sdp/movemeet/utility/LocationFetcher.java [hunk 2]
```diff
@@ -31,9 +31,11 @@ public static void fetchLastLocation(SupportMapFragment supportMapFragment, Fuse
         task.addOnSuccessListener(new OnSuccessListener<Location>() {
             @Override
             public void onSuccess(Location location) {
-                currentLocation = location;
-                Toast.makeText(supportMapFragment.getActivity(), currentLocation.getLatitude() + ", " + currentLocation.getLongitude(), Toast.LENGTH_SHORT).show();
-                supportMapFragment.getMapAsync(callback);
+                if (location != null) {
```
Comment (kotleta2007) [inline app/src/main/java/com/sdp/movemeet/utility/LocationFetcher.java:34]: What's the purpose of this change?

File: app/src/main/java/com/sdp/movemeet/Sport.java [hunk 3]
```diff
@@ -5,5 +5,6 @@
     Soccer,
     Swimming,
     Tennis,
-    Badminton
+    Badminton,
+    Windsurfing
```
Comment (Antho1426) [inline app/src/main/java/com/sdp/movemeet/Sport.java]: Seems like I was not the only one to have to add the "Windsurfing" activity ^^'

**Review by Antho1426 — APPROVED** [review]

File: app/src/androidTest/java/com/sdp/movemeet/HomeScreen/HomeScreenActivityTest.java (hunk 1 above)
Comment (bitshrine) [reply app/src/androidTest/java/com/sdp/movemeet/HomeScreen/HomeScreenActivityTest.java:33]: I wanted to launch the Homescreen myself after having signed in or out to make sure that it would pass a potential authentification check when the Homescreen activity was created.

**Review by kotleta2007 — APPROVED** [review]

File: app/src/androidTest/java/com/sdp/movemeet/Activity/UploadActivityActivityTest.java [hunk 4]
```diff
@@ -202,11 +208,52 @@ public void onComplete(@NonNull Task<AuthResult> task) {
...
         }); */
 
-        mAuth.signOut();
+        //mAuth.signOut();
         scenario.close();
 
     }
 
+    @After
+    public void deleteAndSignOut() {
```
Comment (jlstoupy98) [inline app/src/androidTest/java/com/sdp/movemeet/Activity/UploadActivityActivityTest.java:218]: A clean way to disconnect after the test! well done !

**Review by RoxaneBurri — APPROVED** [review]

File: app/src/main/java/com/sdp/movemeet/utility/LocationFetcher.java (hunk 2 above)
Comment (bitshrine) [reply app/src/main/java/com/sdp/movemeet/utility/LocationFetcher.java:34]: For some reason, I would get errors when running tests, because the location returned by the successful task was somehow null

File: app/src/main/java/com/sdp/movemeet/Sport.java (hunk 3 above)
Comment (bitshrine) [reply app/src/main/java/com/sdp/movemeet/Sport.java]: Oh! I wonder why only that one was required, considering many were added at once.

File: app/src/main/java/com/sdp/movemeet/utility/LocationFetcher.java (hunk 2 above)
Comment (kotleta2007) [reply app/src/main/java/com/sdp/movemeet/utility/LocationFetcher.java:34]: Ok, that makes sense!
